MIP61/MIP61.md

```diff
@@ -93,17 +93,25 @@ The formula is `C*min(1,MKR^q/T^q)`.
 
 #### Action Plan
 
-Rates of payment need recalculation only when the amount of delegated MKR changes. A rough approximation is obtained by recalculating only at regular periods (daily or weekly). Governance Facilitators shall look for changes at least daily and include the DAI transfers as part of an executive vote once per month.
+Rates of payment need recalculation only when the amount of delegated MKR changes. A rough approximation is obtained by recalculating only at regular periods (daily or weekly). Governance Facilitators shall look for changes at least daily and include the DAI transfers as part of an Executive Vote once per month.
 
-#### Compensation Eligibility
+It is expected that Recognized Delegates will confirm both the amount of compensation being paid, and the receiving address with the Governance Facilitators in the week leading up to the Executive Vote, if this is not possible for an individual Recognized Delegate, that individual's compensation will not be included in the Executive Vote and instead will be included in a future Executive Vote at a mutually agreeable time.
```

Comment:
Can we separate this component into separate components? It's now covering quite a lot of stuff.

Reply:
Like, the formula, compensation, and payout process should just be separated.

Reply:
Separated out the payment process